src/common/hexchat.c

```diff
@@ -883,9 +883,9 @@ xchat_init (void)
 		"NAME ENDSUB\n"				"CMD \n\n",
 
 		_("_Open Dialog Window"), "gtk-go-up",
-		_("_Send a File"), "gtk-floppy",
+		_("_Send a File..."), "gtk-floppy",
```

Comment:
Ok, even though it's independent characters everywhere else? Should I also change the others?